api/src/chat/localization/RcdaTextEnglish.ts

```diff
@@ -37,6 +39,11 @@ export class RcdaMyanmarTextEnglish {
     askTownshipName = "What township are you reporting on?";
     askDisasterType = "What is the disaster type?";
     askGeographicalSettingType = "What is the setting?";
+    tellCurrentAdminStack = (currentAdminStack: string) => `The current admin stack selected is - **${this.townships[currentAdminStack]}**. Do you want to change it?`;
```

Comment:
Let's not use 'this' in here, since the semantics of this are so complicated in JS. In other cases like this, I pass in the admin stack name pre-translated, so for example: localizer.mm.tellCurrentAdminStack(localizer.mm.townships[townshipId])

Reply:
Be sure to catch other instances of this as well.